src/py/mains/run.py

```diff
@@ -87,6 +89,9 @@ def wait_for_file(filename, task, job_name = ""):
             if support.job_type == "SGE":
                 fc_run_logger.info( "deleting the job by `qdel` now..." )
                 os.system("qdel %s" % job_name) # Failure is ok.
+            if support.job_type == "SLURM"
+            	fc_run_logger.info( "Deleting the job by 'scancel' now...")
```

Comment:
Haha, no worries Chris, this one was my fault. I normally edit in eclipse but I was having issues committing back to my branch before issuing the pull request so I used the terrible in line text editor Git provides via the HTML website. My apologies for the issue!

-Roberto

Roberto Lleras
Senior Scientist, Bfx. Field Applications Specialist, North America & Europe
Tel: +1.718.578.8987 | Skype: live:roberto.a.lleras_1 | rlleras@pacb.com<mailto:rlleras@pacb.com>
Pacific Biosciences | 1380 Willow Road, Menlo Park, California 94025 | pacb.com<http://www.pacificbiosciences.com/>

From: Christopher Dunn [mailto:notifications@github.com]
Sent: Monday, August 31, 2015 1:27 PM
To: PacificBiosciences/FALCON
Cc: Roberto Lleras
Subject: Re: [FALCON] Fix support for FALCON 0.3 running on SLURM (#181)


In src/py/mains/run.py<https://github.com/PacificBiosciences/FALCON/pull/181#discussion_r38335548>:

> @@ -87,6 +89,9 @@ def wait_for_file(filename, task, job_name = ""):

>              if support.job_type == "SGE":

>                  fc_run_logger.info( "deleting the job by `qdel` now..." )

>                  os.system("qdel %s" % job_name) # Failure is ok.

> +            if support.job_type == "SLURM"

> +             fc_run_logger.info( "Deleting the job by 'scancel' now...")

Do not mix spaces with tabs. What crappy editor are you using? Serious question. Don't be embarrassed. I want to know so that I can tell people what settings to use.

This was also a common problem at Amazon, mostly caused by Eclipse, but the more sophisticated teams included an Eclipse format file with their Java source-code. In Go-lang, people simply run goformat, which is a wonderful tool! All languages should include an auto-formatter.

Unfortunately, tabs ARE considered equivalent to spaces -- specifically, eight spaces, as the docs say quite clearly. Never mixing tabs and spaces is best-practice, but you need to run Python with -t (or -tt) to get warnings (or errors) for violations of said best practice. – Alex Martelli Jan 9 '10 at 19:12

I plan to add -t (and eventually -tt) to catch this sooner.

—
Reply to this email directly or view it on GitHub<https://github.com/PacificBiosciences/FALCON/pull/181/files#r38335548>.